src/thunderbots/software/ai/navigator/obstacle/obstacle.cpp

```diff
@@ -0,0 +1,117 @@
+
+#include "obstacle.h"
+
+const Polygon& Obstacle::getBoundaryPolygon() const
+{
+    return _polygon;
+}
+
+Obstacle::Obstacle() : _polygon{Point(0, 0), Point(0, 1), Point(1, 1), Point(1, 0)} {}
```

Comment:
Hmm, not sure I totally agree with having a default value here, mostly just because I can't think of any reasonable "default" polygon. I'll leave this up to your discretion though.

Reply:
I can remove it